src/main/java/hudson/plugins/jacoco/JacocoBuildAction.java

```diff
@@ -202,12 +196,12 @@ public Object getTarget() {
 	}
 
 	@Override
-	public AbstractBuild<?,?> getBuild() {
+	public Run<?,?> getBuild() {
```

Comment:
This and similar changes are not binary compatible, but I could not find [anything in @jenkinsci referring to code in this plugin](https://github.com/search?utf8=%E2%9C%93&q=user%3Ajenkinsci+-repo%3Ajenkinsci%2Fjacoco-plugin+jacoco&type=Code&ref=searchresults) so probably it is fine.